data/presets/amenity/post_box/post_box_gb.json

```
@@ -0,0 +1,45 @@
+{
+    "icon": "temaki-post_box",
+    "locationSet": {
+        "include": [
+            "gb"
+        ]
+    },
+    "fields": [
+        "operator",
+        "collection_times",
+        "post_box:type",
+        "royal_cypher",
+        "ref"
+    ],
+    "moreFields": [
+        "colour",
+        "indoor",
+        "level",
+        "material",
+        "support",
+        "wheelchair"
+    ],
+    "geometry": [
+        "point",
+        "vertex"
+    ],
+    "tags": {
+        "amenity": "post_box"
+    },
+    "terms": [
+        "letter drop",
+        "mail box",
+        "mail collection box",
+        "mail drop",
+        "mail dropoff",
+        "mailbox",
+        "package drop",
+        "pillar box",
+        "pillarbox",
+        "post box",
+        "postal box",
+        "postbox"
+    ],
+    "name": "Post Box"
```

It is possible to cross reference strings between presets, in order to avoid duplicates for translators:

```suggestion
    "name": "{amenity/post_box}"
```